presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -937,4 +942,62 @@ public HiveClientConfig setHiveMetastoreClientKeytab(String hiveMetastoreClientK
         this.hiveMetastoreClientKeytab = hiveMetastoreClientKeytab;
         return this;
     }
+
+    public enum HdfsAuthenticationType
+    {
+        SIMPLE,
```

Comment:
Same as the metatstore commit.  I would call this `NONE` and the implementations `NoSuchAndSuchAuthentication`